src/toil/provisioners/clusterScaler.py

```diff
@@ -659,23 +732,30 @@ def _removeNodes(self, nodeToNodeInfo, instance_type, numNodes, preemptable=Fals
                 # will be terminated in _removeIgnoredNodes later on
                 # once all jobs have finished, but they will be ignored by
                 # the batch system and cluster scaler from now on
-                nodesToTerminate = [(node,nodeInfo) for (node,nodeInfo) in nodesToTerminate if nodeInfo is not None and nodeInfo.workers < 1]
-            nodesToTerminate = {node:nodeInfo for (node, nodeInfo) in nodesToTerminate}
-            nodeToNodeInfo = nodesToTerminate
+                nodesToTerminate = [
+                    (node, nodeInfo)
+                    for (node, nodeInfo) in nodesToTerminate
+                    if nodeInfo is not None and nodeInfo.workers < 1
+                ]
+            nodesToTerminate = {node: nodeInfo for (node, nodeInfo) in nodesToTerminate}  # type: ignore[assignment]  # FIXME
+            nodeToNodeInfo = nodesToTerminate  # type: ignore[assignment]  # FIXME
```

Comment:
Help, these types don't make sense to me

Reply:
Hmmm... not sure about these.  Will take a look.